scripts/NodeAdmin.Dockerfile

```diff
@@ -27,7 +27,7 @@ COPY . .
 
 RUN yarn run build-node
 
-FROM nginx:1.25.2@sha256:48a84a0728cab8ac558f48796f901f6d31d287101bc8b317683678125e0d2d35 AS runtime
+FROM nginx:stable AS runtime
```

Comment:
lets use a specific stable version here that works, so it will not get damaged again